delta/delta_1.5.6_kek_model_update_2008_2019.sql

```diff
@@ -0,0 +1,424 @@
+COMMENT ON COLUMN qgep_od.damage_manhole.quantification1 IS 'yyy_Quantifizierung 1 gemäss SN EN 13508. Zulässige Eingaben sind in Kapitel 3.1.5 beschrieben. Als Textattribut umgesetzt. / Quantifizierung 1 gemäss SN EN 13508. Zulässige Eingaben sind in Kapitel 3.1.5 beschrieben. Als Textattribut umgesetzt. / Quantification 1 selon la SN EN 13508. Les entrées autorisées sont décrites dans le chapitre 3.1.5. Type texte.';
+
+ALTER TABLE qgep_od.damage_manhole ADD COLUMN quantification2 varchar(20) ;
+COMMENT ON COLUMN qgep_od.damage_manhole.quantification2 IS 'yyy_Quantifizierung 1 gemäss SN EN 13508. Zulässige Eingaben sind in Kapitel 3.1.5 beschrieben. Als Textattribut umgesetzt. / Quantifizierung 1 gemäss SN EN 13508. Zulässige Eingaben sind in Kapitel 3.1.5 beschrieben. Als Textattribut umgesetzt. / Quantification 1 selon la SN EN 13508. Les entrées autorisées sont décrites dans le chapitre 3.1.5. Type texte.';
+
+ ALTER TABLE qgep_od.damage_manhole ADD COLUMN damage_begin  smallint ;
+COMMENT ON COLUMN qgep_od.damage_manhole.damage_begin IS 'yyy_Lage am Umfang: Beginn des Schadens. Werte und Vorgehen sind unter Absatz 3.1.6 genau beschrieben. / Lage am Umfang: Beginn des Schadens. Werte und Vorgehen sind unter Absatz 3.1.6 genau beschrieben. / Emplacement circonférentiel: Début du dommage. Valeurs et procédure sont décrites en détail dans le paragraphe 3.1.6.';
+ ALTER TABLE qgep_od.damage_manhole ADD COLUMN damage_end  smallint ;
+COMMENT ON COLUMN qgep_od.damage_manhole.damage_end IS 'yyy_Lage am Umfang: Beginn des Schadens. Werte und Vorgehen sind unter Absatz 3.1.6 genau beschrieben. / Lage am Umfang: Beginn des Schadens. Werte und Vorgehen sind unter Absatz 3.1.6 genau beschrieben. / Emplacement circonférentiel: Fin du dommage. Valeurs et procédure sont décrites en détail dans le paragraphe 3.1.6.';
+
+ ALTER TABLE qgep_od.damage_manhole ADD COLUMN distance  decimal(7,2) ;
+COMMENT ON COLUMN qgep_od.damage_manhole.distance IS 'yyy_Länge von Rohranfang bis zur Feststellung bzw. Länge ab Oberkante Deckel (siehe Richtlinie Absatz 3.1.1.) in m mit zwei Nachkommastellen / Länge von Rohranfang bis zur Feststellung bzw. Länge ab Oberkante Deckel (siehe Richtlinie Absatz 3.1.1.) in m mit zwei Nachkommastellen / Longueur entre le début de la canalisation resp. longueur entre le bord supérieur du couvercle et l’observation (cf. paragraphe 3.1.1.) en m avec deux chiffres après la virgule.';
+
+-- Copy data from Superclass to Subclass
+
+UPDATE qgep_od.damage_manhole dm
+SET 
+	quantification1 = dg.quantification1,
+	quantification2 = dg.quantification2,
+	damage_begin = dg.damage_begin,
+	damage_end = dg.damage_end,
+	distance = dg.distance
+	
+FROM qgep_od.damage dg
+WHERE  dm.obj_id = dg.obj_id; 
+
+
+-- delete columns in qgep_od.damage
+
+ALTER TABLE qgep_od.damage
+DROP COLUMN IF EXISTS quantification1,
+DROP COLUMN IF EXISTS quantification2,
+DROP COLUMN IF EXISTS damage_begin,
+DROP COLUMN IF EXISTS damage_end,
+DROP COLUMN IF EXISTS distance;
```

Comment:
These changes will require adapting the export script (not too difficult)

Are there other parts of QGEP that are affected by that (e.g. forms in the QGIS project) ?

Reply:
I checked the project and found the following things that might be affected:

- [x] [vw_damage.yaml](https://github.com/QGEP/datamodel/blob/master/view/vw_damage.yaml) - it takes all attributes from super and subclass without specifying attribute names - so no problem
- [x] [vw_vw_maintenance_examination.yaml.yaml](https://github.com/QGEP/datamodel/blob/master/view/vw_maintenance_examination.yaml.yaml) - it takes all attributes from super and subclass without specifying attribute names - so no problem
- [ ] whether this influences the form for maintenance connected to it I do not know yet
- [ ] As you mentioned I have to change [04_vsa_kek_extension.sql](https://github.com/QGEP/datamodel/blob/master/04_vsa_kek_extension.sql) and
- [ ] [09_qgep_dictionaries_kek.sql](https://github.com/QGEP/datamodel/blob/master/09_qgep_dictionaries_kek.sql)
- [ ] I think then may be some of these fixes will not be needed anymore: https://github.com/QGEP/datamodel/tree/master/fixes
- [ ] if you see other points please add

Reply:
I hope to have a new version for 04_vsa_kek_extension.sql and 09_qgep_dictionaries_kek.sql till tomorrow ready.

Reply:
@olivierdalang I have pushed a new version of 04_vsa_kek_extension.sql and 09_qgep_dictionaries_kek.sql